pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -265,15 +265,15 @@ func (r *ReconcileWindowsMachineConfig) createWindowsWorkerNodes(count int) bool
 			errs = append(errs, errors.Wrap(err, "error creating windows VM"))
 			log.Error(err, "error creating windows VM")
 		}
-		log.V(5).Info(fmt.Sprintf("created the Windows VM: %s",
-			createdVM.GetCredentials().GetInstanceId()))
+		log.V(5).Info("created the Windows VM", "instance",
```

Comment:
I think as per recommendation we should have 0 or 1?

As per [docs](https://github.com/kubernetes-sigs/controller-runtime/blob/master/TMP-LOGGING.md)

`While it's possible to use higher log levels, it's recommended that you stick with V(1) or V(0)(which is equivalent to not specifyingV), and then filter later based on key-value pairs or messages; different numbers tend to lose meaning easily over time, and you'll be left wondering why particular logs lines are at V(5)instead ofV(7)`.`

Reply:
Checking to see if V(1) can work as we expect by specifying the log level.

Reply:
Since operator-sdk uses zap https://github.com/kubernetes-sigs/controller-runtime/tree/master/pkg/log/zap for logging. We can view the debug logs by setting a flag --zap-level=1. Therefore set the above log to v(1)

Reply:
Thank you for verifying this.